_events/extra-life.md

```diff
@@ -9,32 +9,36 @@ Extra Life unites thousands of gamers around the world to play games in support
 
 # Extra Life at Michigan Tech
 
-Extra Life at Michigan Tech is not your standard 24 hour gaming marathon. At this 6 hour event, we've chosen to create a unique and exciting gaming environment where you can have fun and hang out with your fellow Huskies. Attendee's are encouraged to bring their own games, consoles, and even computers. Some games and consoles will be provided by Triangle Fraternity Brothers and other organizations for open play. In support of [Children's Miracle Network Hospitals](https://childrensmiraclenetworkhospitals.org/), please consider donating at our event or on out [fundraising page](https://www.extra-life.org/participant/MTU_Extra_Life).
+Extra Life at Michigan Tech is not your standard 24 hour gaming marathon. At this 6 hour event, we've chosen to create a unique and exciting gaming environment where you can have fun and hang out with your fellow Huskies. Attendee's are encouraged to bring their own games, consoles, and even computers. Some games and consoles will be provided by Triangle Fraternity Brothers and other organizations for open play. In support of [Children's Miracle Network Hospitals](https://childrensmiraclenetworkhospitals.org/), please consider donating at our event or on our [fundraising page](https://www.extra-life.org/participant/MTU_Extra_Life).
+
+## April 7th, 2018
+## 8 PM - 2 AM
+## Wadsworth Hall Dining Hall
 
 {% include image-section.html image="/assets/img/extra-life/bluegridpattern_icons.jpg" content='
```

Comment:
Please replace the `'` for the `content` with a `"` to have CI pass.

Reply:
how does this even happen. I didn't touch that part. lol